huggingface/pytorch/training/docker/1.6/py3/cu110/Dockerfile.gpu

```diff
@@ -14,8 +14,7 @@ RUN pip install --no-cache-dir \
 	datasets==${DATASETS_VERSION}
 RUN apt-get update \
  && apt install git-lfs \
-   	unzip \
- && apt-get clean \  
+ && apt-get clean \
```

Comment:
Why remove `unzip` here?

Reply:
@SergTogul told that in newly released pytorch base images has `unzip`.  With `unzip` here, pytorch build fails.